tests/std/tests/P1502R1_standard_library_header_units/custombuild.pl

```diff
@@ -88,18 +88,19 @@ ()
         "version",
     );
 
+    my $export_header_options = "/exportHeader /Fo /MP";
     my $header_unit_options = "";
 
     foreach (@stl_headers) {
+        $export_header_options .= " $stl_include_dir/$_";
+
         $header_unit_options .= " /headerUnit";
         $header_unit_options .= " $stl_include_dir/$_=$_.ifc";
         $header_unit_options .= " $_.obj";
-
-        # TRANSITION, remove /DMSVC_INTERNAL_TESTING after all compiler bugs are fixed
-        Run::ExecuteCL("/DMSVC_INTERNAL_TESTING /exportHeader \"<$_>\" /Fo$_.obj");
     }
 
     # TRANSITION, remove /DMSVC_INTERNAL_TESTING after all compiler bugs are fixed

```

Comment:
"after all compiler bugs are fixed" could be a _very_ long time. Maybe this should say "after the relevant compiler bugs are fixed"? =) (No change requested - I'm just being cheeky.)